docker/docker-compose.yml

```diff
@@ -215,6 +215,24 @@ services:
       - pds
     command: "run /postman/postman-collection.json --env-var baseUrl=${REG_API_URL}"
 
+  # Executes Registry API integration tests as a Postman collection and reports the results into JPL TestRail
+  testrail-reporting-test:
+    profiles: []
```

Comment:
Is this not included in any of the docker compose profiles? I would like to suggest to include this in a profile (even if this is going to be the only one thing included in the profile) to make it consistent with other docker compose profile based executions.